src/game/server/entities/player.h

```diff
@@ -449,6 +455,9 @@ class CBasePlayer : public CBaseMonster
 	//For saving and level changes.
 	int m_HudColor = RGB_HUD_COLOR.ToInteger();
 
+	bool m_bInfiniteAir;
```

Comment:
These should be save/restored to match the behavior of `FL_GODMODE` and other cheat flags.

Reply:
Added both variables to the save/restore table.